src/Tribe/Tickets.php

```diff
@@ -477,7 +477,7 @@ public function get_ticket_move_link( $post_id, $ticket = null ) {
 				return;
 			}
 
-			$button_text = ( 'Tribe__Tickets__RSVP' === $ticket->provider_class ) ? __( 'Move RSVP', 'event-tickets' ) : __( 'Move Ticket', 'event-tickets' ) ;
+			$button_text = ( 'Tribe__Tickets__RSVP' === $ticket->provider_class ) ? sprintf( __( 'Move %s', 'event-tickets' ), tribe_get_rsvp_label_singular( 'move_ticket_button_text' ) ) : sprintf( __( 'Move %s', 'event-tickets' ), tribe_get_ticket_label_singular( 'move_ticket_button_text' ) ) ;
```

Comment:
Can you adjust the sprintf below in `$move_link` to properly pass in `esc_html( $button_text )` as a true placeholder (`%2$s`)?

Reply:
done